sync/sync-impl/src/main/java/com/duckduckgo/sync/impl/engine/RealSyncEngine.kt

```diff
@@ -187,8 +187,8 @@ class RealSyncEngine @Inject constructor(
 
             is Success -> {
                 Timber.d("Sync-Feature: get success")
-                syncStateRepository.updateSyncState(SUCCESS)
                 persistChanges(result.data, conflictResolution)
+                syncStateRepository.updateSyncState(SUCCESS)
```

Comment:
Why the re-order. If order is important here, I think it'd merit a comment as to **why**